python/ql/test/query-tests/Functions/general/functions_test.py

```diff
@@ -36,8 +36,8 @@ def ok3(x):
 def ok4(x = []):
     return len(x)
 
-def mpd(x = []):
-    x.append("x")
+
+
```

Comment:
You might consider removing these lines..

Reply:
Ah yes. I left them in so as to not have to change a bunch of `.expected` files (since they all refer to `functions_test.py`), but I guess now that we're done with the review, I can clean the file up a bit.